binr/blob/Makefile

```diff
@@ -90,7 +91,7 @@ build:
 		rm -f _$${FILE} ; \
 	done
 	rm -f _*
-	${CC} -o radare2 ${SOURCES} ${CFLAGS} ${LDFLAGS}
+	${CC} -I$(SHLR)/sdb/src -o radare2 ${SOURCES} ${CFLAGS} ${LDFLAGS}
```

Comment:
this should not be necessary I think, because it should already be in CFLAGS after you include sdb.mk